app/controller/partycontroller.ctrl.php

```diff
@@ -201,10 +201,29 @@ public function create()
                         }
 
                         if(hasRole($this->user, 'Host')){
+                            // send email to Admin
+                              $message = "<p>Hi,</p>" .
+                                       "<p>This is an automatic email to let you know that ". $name . " has created a party on the <strong>" . APPNAME . "</strong>.</p>" .
```

Comment:
Where is this `$name` variable defined?
This should come from $this->user->name I think.

Reply:
Its defined in usercontroller.

Reply:
It won't be accessible here - you will need to use the `$this->user` object.